portal-frontend/src/app/services/toast/toast.service.ts

```diff
@@ -0,0 +1,32 @@
+import { Injectable } from '@angular/core';
+import { MatSnackBar } from '@angular/material/snack-bar';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class ToastService {
+  constructor(private snackBar: MatSnackBar) {}
+
+  private TOAST_DURATION = 3000;
```

Comment:
Not in the scope of this ticket. I do not remember for sure, but I think, based on WCAG these durations must be adjustable by a user.